server/lib/url.js

```diff
@@ -29,19 +30,21 @@ module.exports.external = uuid => {
 };
 
 module.exports.stream = (metadatum, options) => {
-	const streamUrl = `http://www.ft.com/stream/${metadatum.taxonomy}Id/${metadatum.idV1}`;
+	const fetch = wrapFetch(nodeFetch, {
+		tag: 'url.stream',
+	});
 
-	if(process.env.VERIFY_STREAM_URLS !== 'true') {
-		return Promise.resolve(streamUrl);
-	}
-
-	const headers = {
-		// Ensure we're opted-in to Next
-		cookie: 'FT_SITE=NEXT',
-	};
+	const url = 'http://next-es-interface.ft.com/things?authority=http://api.ft.com/system/FT-TME' +
```

Comment:
Agreed, it's one of the un-authed endpoints (https://github.com/Financial-Times/next-es-interface/blob/master/server/app.js#L73-L82), but not documented. I'll add a link to this code, and request that it's documented.